jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -250,6 +243,47 @@ public Bpmn2JsonUnmarshaller() {
         }
     }
 
+    private static String escapeXmlString(String string) {
```

Comment:
I think this method could be implemented just with this third party library:
```java
org.apache.commons.lang.StringEscapeUtils.escapeXml(string);
```

Reply:
@jomarko yes, i will create a new jira for this if thats ok with you because changing could cause unaware issues and i rather deal with that separately please.

Reply:
@tsurdilo could you send me a link for that jira please?

Reply:
@jomarko sorry, sure - i created https://issues.jboss.org/browse/JBPM-6599. thanks.